packages/components-sketch/views/card-static.hbs

```diff
@@ -9,38 +9,41 @@ file, You can obtain one at https://mozilla.org/MPL/2.0/.
 --}}
 
 <div data-sketch-artboard="Card Static" class="artboard">
-    <h1 class="title--artboard">Card Static</h1>
-    
-    <h2 class="title--artboard-section">Padding on</h2>
-    <div class="agent-states--grid">
-        <div class="agent-states--item">
-            <p class="agent-states--label">:enabled</p>
-            <scale-card data-sketch-key="card-static-1">
-                <img src="/placeholder.png" alt="">
-            </scale-card>
-        </div>
+  <h1 class="title--artboard">Card Static</h1>
+
+  <h2 class="title--artboard-section">Padding on</h2>
+  <div class="agent-states--grid">
+    <div class="agent-states--item">
+      <p class="agent-states--label">:enabled</p>
+      <scale-card data-sketch-key="card-static-1">
+        <img src="/placeholder.png" alt="" />
+      </scale-card>
     </div>
+  </div>
 
-    <h2 class="title--artboard-section">Padding off</h2>
-    <div class="agent-states--grid">
-        <div class="agent-states--item">
-            <p class="agent-states--label">:enabled</p>
-            <scale-card data-no-padding data-sketch-key="card-static-2">
-                <img src="/placeholder.png" alt="" style="display: block">
-            </scale-card>
-            <script>
-                /* I couldn't find a way to do this with "theming" */
-                document.querySelector('[data-no-padding]').styles = { card__body: { padding: 0 } };
-            </script>
-        </div>
+  <h2 class="title--artboard-section">Padding off</h2>
+  <div class="agent-states--grid">
+    <div class="agent-states--item">
+      <p class="agent-states--label">:enabled</p>
+      <scale-card data-no-padding data-sketch-key="card-static-2">
+        <img src="/placeholder.png" alt="" style="display: block" />
+      </scale-card>
+      <script>
+        /* I couldn't find a way to do this with "theming" */
+        document.querySelector('[data-no-padding]').styles = {
```

Comment:
is there a way of getting rid of `styles`?

Reply:
yeah, we need to take a look at the requirements for this
this is also related to #344

Reply:
it this a blocker then?